sentry/src/main/java/io/sentry/IScopes.java

```diff
@@ -414,6 +423,13 @@ default void configureScope(@NotNull ScopeCallback callback) {
    */
   public @NotNull IScope getIsolationScope();
```

Comment:
Does not need to be `public` in interface.
Still in discussion if we actually need it and whether to mark it as `@ApiStatus.Internal`